src/queue-list/laboratory-tabs.component.tsx

```diff
@@ -23,6 +24,10 @@ enum TabTypes {
   ALL,
 }
 
+// The lab-panels-slot will not be visible in the implementer tools UI when used below
+// in the value for ComponentContext.Provider as the UI viewer depends on the div that
+// cannot be added because of how the Carbon components work. Implementers can add more
+// tab extensions to this slot via the routes.js file configuration.
```

Comment:
Do we need this comment? or is this in relation to why u moved this into the main return?

Reply:
have removed it